onebusaway-android/src/main/java/org/onebusaway/android/report/ui/SimpleArrivalListFragment.java

```diff
@@ -273,6 +279,6 @@ private ArrivalsListLoader getArrivalsLoader() {
     }
 
     public void setCallback(Callback callback) {
-        this.callback = callback;
+        this.mCallback = callback;
```

Comment:
This is for `onArrivalItemClicked` event defined in `SimpleArrivalListFragment`. I think this is necessary. Because We are setting this callback from its parent activity: https://github.com/cagryInside/onebusaway-android/blob/master_open311_prp/onebusaway-android/src/main/java/org/onebusaway/android/report/ui/SimpleArrivalListFragment.java#L83